packages/modules/imodel-browser/package.json

```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "@itwin/itwinui-icons-react": "^1.2.0",
-    "@itwin/itwinui-react": "^1.11.0",
+    "@itwin/itwinui-react": "^1.35.0",
```

Comment:
have you verified that itwinui update has not broken anything in other packages? Might be tricky with this one..

Reply:
Checked in storybook as much as I could - it looks like nothing is broken. Also, all tests passed too.